_drafts/status-update.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Put up the PR directly as _posts/2020-02-07-weekly-update.md - no need to back-and-forth; merge=release!